savant/deepstream/meta/frame.py

```diff
@@ -48,6 +52,24 @@ def objects_number(self) -> int:
         """
         return self.frame_meta.num_obj_meta
 
+    @property
+    def tags(self) -> Dict[str, Any]:
+        """Returns tags of frame. These tags are part of the meta information about
+        the frame that comes with the frames in the module.
+
+        :return: Dictionary with tags
+        """
+        if self.gst_frame_meta is None:
```

Comment:
We can single out the lazy load of gst_frame_meta and use it in the future for other source frame properties in the future. Up to you.